event_routing_backends/processors/caliper/event_transformers/enrollment_events.py

```diff
@@ -30,6 +34,8 @@ def get_action(self):
         """
         if self.event['name'] == 'edx.course.enrollment.activated':
             return 'Activated'
+        if self.event['name'] == 'edx.course.completed':
```

Comment:
we should get event data via `get_data` accessor method.

Reply:
Done